README.md

```diff
@@ -2,19 +2,19 @@
   <img alt="githubdark-logo" src="https://rawgit.com/StylishThemes/logos/master/github.dark/githubdark-mini.svg" width="580">
   <br>
   <a href="https://github.com/StylishThemes/GitHub-Dark/tags">
-    <img src="https://img.shields.io/github/tag/StylishThemes/GitHub-Dark.svg?label=tag" alt="Tag">
+    <img src="https://img.shields.io/github/tag/StylishThemes/GitHub-Dark.svg?label=tag&style=for-the-badge" alt="Tag">
   </a>
   <a href="https://github.com/StylishThemes/GitHub-Dark/stargazers">
-    <img src="http://github-svg-buttons.herokuapp.com/star.svg?user=StylishThemes&repo=GitHub-Dark&style=flat&background=007ec6" alt="Star">
+    <img src="https://img.shields.io/github/stars/StylishThemes/GitHub-Dark.svg?label=Stars&style=for-the-badge" alt="Stars">
   </a>
-  <a href="http://github.com/StylishThemes/GitHub-Dark/fork">
-    <img src="http://github-svg-buttons.herokuapp.com/fork.svg?user=StylishThemes&repo=GitHub-Dark&style=flat&background=007ec6" alt="Fork">
+  <a href="https://github.com/StylishThemes/GitHub-Dark/network">
+    <img src="https://img.shields.io/github/forks/StylishThemes/GitHub-Dark.svg?style=for-the-badge" alt="Fork">
   </a>
   <a href="https://david-dm.org/StylishThemes/GitHub-Dark?type=dev">
-    <img src="https://img.shields.io/david/dev/StylishThemes/GitHub-Dark.svg?label=%20devDependencies%20" alt="devDependencies">
+    <img src="https://img.shields.io/david/dev/StylishThemes/GitHub-Dark.svg?label=devDependencies&style=for-the-badge" alt="devDependencies">
   </a>
   <a href="https://gitter.im/StylishThemes/GitHub-Dark">
-    <img src="https://img.shields.io/gitter/room/StylishThemes/Github-Dark.js.svg?maxAge=2592000" alt="Gitter">
+    <img src="https://img.shields.io/gitter/room/StylishThemes/Github-Dark.js.svg?maxAge=2592000&style=for-the-badge" alt="Gitter">
```

Comment:
I think I prefer the old (smaller) badges.

Reply:
Let me try the smaller flat-square instead, but sure I thought these may not be that attractive at first look.

Reply:
getting rid of my badges, huh :) @the-j0k3r

Reply:
I'd prefer to leave the badges as-is...

Reply:
@ddavison actually I would prefer your badge for starring, but your styles are limited to two styles only, could you add more?.

@Mottie sure, OK, I only tried to make them all look the same instead of a mix match of different styles